website/docs/concepts/router.mdx

```diff
@@ -45,7 +45,9 @@ matched, the router navigates to the path with `not_found` attribute. If no rout
 a message is logged to console stating that no route was matched.
 
 Finally, you need to register one of the Router context provider components like `<BrowserRouter />`.
-It provides location information and navigator to its children.
+It provides location information and navigator to its children. I.e. certain components depend on the
```

Comment:
```suggestion
It provides location information and navigator to its children. i.e. certain components depend on the
```

See https://grammar.yourdictionary.com/grammar/style-and-usage/how-to-use-i-e-and-e-g.html

Reply:
I don't think I'm miss using i.e. and e.g.

I'm using them pretty literally, but I could also not fully understand them. I'll just in line them to what I mean them to be

i.e. == that is
e.g. == for example